types/jcof/jcof-tests.ts

```diff
@@ -0,0 +1,13 @@
+import jcof = require('jcof');
+
+const exampleObject = {
+	people: [
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/jcof/jcof-tests.ts;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.